workspace/src/app/views/pages/Project/FileWidget/FileWidget.tsx

```diff
@@ -93,9 +93,9 @@ export const FileWidget = () => {
                     </CardTitle>
                     <CardOptions>
                         <IconButton
-                            name="item-add-artefact"
+                            name="item-upload"
                             data-test-id="project-files-widget-add-file-btn"
-                            text={t("common.action.AddSmth", { smth: t("widget.FileWidget.file") })}
+                            title={t("FileUploader.modalTitle", "Upload file")}
```

Comment:
@arausly Why did you changes from tooltip element (via `text`) to a html title (via `title`)?